.github/workflows/semgrep.yml

```
@@ -0,0 +1,35 @@
+      contents: read
+      security-events: write
+    runs-on: ubuntu-latest
+    timeout-minutes: 15
+    steps:
+      - uses: actions/checkout@v4
+      - uses: semgrep/semgrep-action@v1
+        id: semgrep
+        with:
+          config: p/ci
+          generateSarif: true
+          publishToken: ${{ github.event_name == 'pull_request' && github.event.pull_request.head.repo.fork ? '' : secrets.SEMGREP_APP_TOKEN }}
```

actionlint (1.7.7)

could not parse as YAML: yaml: line 22: mapping values are not allowed in this context

(syntax-check)